tests/rules/test_get_branch_0v_to_dash_v.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from thefuck.rules.git_branch_0v_to_dash_v import match, get_new_command
+from thefuck.types import Command
+
+
+@pytest.fixture
+def output():
+    return ""
```

Comment:
This fixture can be removed and all it's occurrences be replaced by `""`.

Reply:
Yes! It's great that you followed patterns, that's a big yes. 👍 

But in that specific case, there's nothing special being done in the fixture. It simply returns an empty string. As you correctly suspected, it's more efficient (and also more readable) to just use the actual value (`""`) instead of the fixture.

Reply:
I tried replacing lines 6-8 with an empty string per your comment and that caused test failures. I tired just commenting out lines 6-8 and that lead to test failures too.

So, being unfamiliar with pytest fixtures, I assume these three lines need to stay in place.

If that is incorrect, please show me the correct way to remove this fixture and replace it with an empty string.